src/Inkluzitron/Data/BotDatabaseContext.cs

```diff
@@ -81,6 +84,13 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             modelBuilder
                 .Entity<VoteReplyRecord>()
                 .HasAlternateKey(nameof(VoteReplyRecord.GuildId), nameof(VoteReplyRecord.ChannelId), nameof(VoteReplyRecord.ReplyId));
+
+            modelBuilder.Entity<Badge>()
+                .Property(i => i.Image)
+                .HasConversion(new ValueConverter<MagickImage, byte[]>(
+                    from => from.ToByteArray(MagickFormat.Png),
```

Comment:
Tohle je sice příjemný způsob jak se k těm obrázkům dostat, ale není nikde zaručeno, že se disposují. Myslím, že to má potenciál leakovat:

1. Natáhneš si uživatele s badgema
2. Nakreslíš jeho $body (třeba)
3. Zahodíš DbContext, ale Image ke kterému jsi přistoupil, žije dál

Možným řešením by bylo udělat si extension metodu, která ti z byte[] vyrobí ten obrázek, a říkat si o to explicitně.
Entity framework by taky mohl umět IDisposable, ale nevím jak moc je OK, když ti "obyčejný" přístup k DB datům pod rukama papá paměť.

Reply:
Předělal jsme to na ten `byte[]`. Extension metoda asi není nutná, protože `MagickImage` konstruktor žere přímo ten `byte[]`.